core/src/filed/fd_plugins.h

```diff
@@ -137,20 +138,22 @@ enum
 };
 
 struct io_pkt {
```

Comment:
There is a lot of code looking like:
```
struct io_pkt io;
memset(&io, 0, sizeof(io));
io.pkt_size = sizeof(io);
io.pkt_end = sizeof(io);
io.filedes = -1;
io.func = IO_SOMEFUNC;
```
Which could be turned into:
```
io_pkt io{IO_SOMEFUNC};
```
We just need to zero-initialize all the members and add a constructor.

Reply:
Done, also did that for other structs